docs/decisions/0008-pull-requests-code-review-guide.md

```diff
@@ -0,0 +1,103 @@
+# Pull Requests and Code Review Guide
+
+This is a guide not law.
+
+Use common sense.
+
+Have fun.
+
+## Protocol
+
+ 1. Open PR from your personal fork to a aquarist-labs repo.
+ 2. Assign to yourself. Set related Issue if it exists.
+ 3. Pick a reviewer. Ideally someone with domain knowledge. Pick more than one if the PR is particularly complex.
+ 4. Wait for review(s). Give about 2 days. If you haven't heard back after that or the PR is urgent ping reviewer(s).
+ 5. Discuss, address feedback until ≥ 1 reviewer(s) approve.
+ 6. Merge as soon as your work is approved
+
+## Review Purpose
+
+Improve quality
+
+Catch bugs
+
+Good engineering
+
+Side effects:
+
+- Learning opportunities
+- Expose where we as a team have blind spots and more work to do (e.g create issues)
+
+## Pull Request Anatomy
+
+Has a *topic*, *description* and ≥ 1 *commits*.
+
+A PR or commit must not contain changes that are not mentioned in a commit message.
+It does what it says it does.
+
+*Topic*
+Which feature, bug fix does the PR address?
+
+*Description*
+Use the provided template and checklist.
+Add issue number if it exists.
+Summarize your changes.
+
+*Commits*
+Each commit contributes to the PR's topic.
+Some exceptions are okay: Refactoring to make follow ups possible/easier. Minor formatting changes.
```

Comment:
Maybe add that individual commits need to be GPG signed and have a DCO?

Reply:
makes sense. adding it in